share/templates/reveal/index.html.j2

```diff
@@ -145,21 +141,21 @@ require(
     },
     [
       "{{ reveal_url_prefix }}/dist/reveal.js",
-      "{{ reveal_url_prefix }}/plugin/notes/notes.js"
+      "{{ reveal_url_prefix }}/plugin/notes/notes.js",
+      "https://cdn.jsdelivr.net/npm/reveal.js-mermaid-plugin@2.2.0/plugin/mermaid/mermaid.js"
```

Comment:
I would recommend hoisting this to a configurable on the python side:

```py
# nbconvert/exporters/slides.py
reveal_url_prefix = Unicode(
  help="..."
).tag(config=True)

reveal_mermaid_js_url = Unicode(
  default="https://cdn.jsdelivr.net/npm/reveal.js-mermaid-plugin@2.2.0/plugin/mermaid/mermaid.js",
  help="...",
).tag(config=True)
```

```js
[
      "{{ reveal_url_prefix }}/dist/reveal.js",
      "{{ reveal_url_prefix }}/plugin/notes/notes.js",
      {% if reveal_mermaid_js_url %}"{{ reveal_mermaid_js_url }}",{% endif %}
]
```

And then below:

```js
plugins: [
  RevealNotes,
  {% if reveal_mermaid_js_url %}RevealMermaid,{% endif %}
]
```

Reply:
ok thanks, makes sense, I did it.

Any further things I can do or should know?

Thanks again,
Franz